docs/reference/analysis/tokenizers/simplepatternsplit-tokenizer.asciidoc

```diff
@@ -0,0 +1,107 @@
+[[analysis-simplepatternsplit-tokenizer]]
+=== Simple Pattern Split Tokenizer
+
+experimental[]
+
+The `simplepatternsplit` tokenizer uses a regular expression to split the
+input into terms at pattern matches. The set of regular expression features it
+supports is more limited than the <<analysis-pattern-tokenizer,`pattern`>>
+tokenizer, but the tokenization is generally faster.
+
+This tokenizer does not produce terms from the matches themselves. To produce
+terms from matches using patterns in the same restricted regular expression
+subset, see the <<analysis-simplepattern-tokenizer,`simplepattern`>>
+tokenizer.
+
+This tokenizer uses http://lucene.apache.org/core//6_5_1/core/org/apache/lucene/util/automaton/RegExp.html[Lucene regular expressions].
+For an explanation of the supported features and syntax, see <<regexp-syntax,Regular Expression Syntax>>.
+
+The default pattern is the empty string, which produces one term containing the
+full input. This tokenizer should always be configured with a non-default
+pattern.
+
+[float]
+=== Configuration
+
+The `simplepatternsplit` tokenizer accepts the following parameters:
+
+[horizontal]
+`pattern`::
+
+  A http://lucene.apache.org/core//6_5_1/core/org/apache/lucene/util/automaton/RegExp.html[Lucene regular expression], defaults to the empty string.
```

Comment:
Do you mean the empty line or the leading whitespace (or both)? Yes, but for no other reason than it was like that in the other asciidoc files I looked at. It seems like the leading whitespace is [the convention](https://github.com/elastic/docs#horizontal-definition-lists). I don't really have a strong opinion about readability here so I'm fine with removing the empty line at least